src/components/contextual/pages/pool/PoolTransactionsCard/PoolTransactionsCard.vue

```diff
@@ -53,21 +61,42 @@ const activeTab = ref(tabs[0].value);
   >
     <BalTabs v-model="activeTab" :tabs="tabs" no-pad class="-mb-px" />
   </div>
-  <Activities
-    v-if="activeTab === PoolTransactionsTab.ALL_ACTIVITY"
-    :pool-activity-type="PoolTransactionsTab.ALL_ACTIVITY"
-    :pool="pool"
-    :loading="loading"
-  />
-  <Activities
-    v-else-if="activeTab === PoolTransactionsTab.USER_ACTIVITY"
-    :pool-activity-type="PoolTransactionsTab.USER_ACTIVITY"
-    :pool="pool"
-    :loading="loading"
-  />
-  <Swaps
-    v-else-if="activeTab === PoolTransactionsTab.SWAPS"
-    :pool="pool"
-    :loading="loading"
-  />
+  <template v-if="isStablePhantomPool">
```

Comment:
for now these are seperate, but in the future its possible to combine them and move the logic inside.